### PR TITLE
Disable buildpayment accountID->username resolution

### PR DESCRIPTION
**Pull request by mlsteele**
Remove built.toUsername. That field was used only for a canned feature where an account ID could morph in the gui into a username.

There is a related bug that will be fixed in an upcoming PR where when the service is told to send to an account ID it actually sends to a user.

### CODE REVIEW

File: shared/wallets/confirm-form/participants/container.js [hunk 1]
```diff
@@ -8,21 +8,20 @@ const mapStateToProps = state => {
...
-  const recipientFullName = userInfo ? userInfo.fullname : ''
+  let recipientFullName = userInfo ? userInfo.fullname : ''
   const fromAccount = getAccount(state, built.from)
 
-  const recipientType = build.recipientType
   let recipientStellarAddress
   let recipientAccountIsDefault
   let recipientAccountName
   let recipientAccountAssets
   if (recipientType === 'keybaseUser') {
```
Comment (buoyad) [inline shared/wallets/confirm-form/participants/container.js]: I think you can remove this block entirely

Comment (mlsteele) [reply shared/wallets/confirm-form/participants/container.js]: which block?

Comment (buoyad) [reply shared/wallets/confirm-form/participants/container.js]: Can replace
```
if (recipientType === 'keybaseUser') {
    if (build.to.includes('@')) {
      // this is an sbs assertion
      recipientFullName = ''
    }
} else
```
with
``` 
if (recipientType !== 'keybaseUser')
```
I don't think we'll ever find a fullName in `userInfo` for sbs, and if we do I don't think it'd be inaccurate to show.

Comment (mlsteele) [reply shared/wallets/confirm-form/participants/container.js]: ah nice thanks

**Pull request merged**